lib/validate.js

```diff
@@ -56,12 +56,7 @@ function isValidatable(real, expected, type, cb) {
   return proxy(getValidatableObject(real, expected, type), 'isValidatable', cb);
 }
 
-function validate(real, expected, type, cb) {
-  return proxy(getValidatableObject(real, expected, type), 'validate', cb);
-}
-
 module.exports = {
-  validate,
```

Comment:
Removed legacy `validate` because we are already using `next/validate` since the last release.